testing/kuttl/e2e/cluster-start/00-assert.yaml

```diff
@@ -17,3 +17,8 @@ metadata:
     postgres-operator.crunchydata.com/pgbackrest-backup: replica-create
 status:
   succeeded: 1
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cluster-start-primary
```

Comment:
So we assert that the service exists because our later test will use that service to connect. Just anecdotally, did the test ever fail for you without this?

Reply:
Nope, it hasn't failed for me without. I think waiting for the initial backup to complete means that the service should always exist.